cli/src/cli.rs

```diff
@@ -27,7 +27,7 @@ use crate::{
 };
 
 const DEFAULT_LOG_LEVEL: &str = "debug";
-const DEFAULT_NODE_URL: &str = "https://api.testnet.shimmer.network";
+const DEFAULT_NODE_URL: &str = "http://localhost:8080";
```

Comment:
intentional?

Reply:
Yeah there is not testnet for 2.0 yet so the only solution is local private tangle, we'll switch back later